Mergin/create_project_wizard.py

```diff
@@ -112,11 +112,27 @@ def initializePage(self):
             self.for_current_proj = False
 
     def populate_namespace_cbo(self):
-        self.project_owner_cbo.addItem(self.parent.username)
-        if self.parent.user_organisations:
-            self.project_owner_cbo.addItems(
-                [o for o in self.parent.user_organisations if self.parent.user_organisations[o] in ["admin", "owner"]]
-            )
+        if self.parent.workspaces is not None:
+            for ws in self.parent.workspaces:
+                try:
```

Comment:
these check are in different file as well, is it possible to have some check function instead?